src/routes/exercise-finished/ExerciseFinishedScreen.tsx

```diff
@@ -20,20 +28,27 @@ const Root = styled.View`
 const UpperSection = styled.View`
```

Comment:
Should be passed as prop rather than accessing global constants:
```suggestion
const UpperSection = styled.View<{ exerciseUnlocked: boolean }>`
```